dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyLister.java

```diff
@@ -33,10 +32,10 @@ public static void main(String[] args) throws RepositoryException {
     
     try {
       DefaultArtifact artifact = new DefaultArtifact(args[0]);
-    
-      DependencyGraph graph =
-          DependencyGraphBuilder.getCompleteDependencies(artifact);
-      
+
+      DependencyGraphBuilder dependencyGraphBuilder = new DependencyGraphBuilder();
```

Comment:
This object can only be used once. The instance state makes it not reusable. I suspect what we really need here is a static method that uses private classes and objects to keep track of the invocation state. However none of that should be exposed in the published API. 

And in any case, that's just an implementation detail. Looking at this, what you have is a static method. Everything that's being stored in the fields could be passed along through method arguments from one method call to the next.